.github/pull_request_template.md

```diff
@@ -7,10 +7,49 @@ The results of these can be either seen under the "Files changed" section of a P
 
 NOTE: Please grant permission for repository maintainers to edit your PR.  It is EXTREMELY common for PRs to be held up due to trivial changes being requested and the author being unavailable to make them. -->
 
+<!-- Your PR title, or how to categorize the changes you made.
+The basics of how we can categorize-organize PRs are the following:
+
+type(scope): short message
+
+- type is mandatory, supported types are listed further down.
+- scope is optional, must be surrounded by parentheses, supported scopes are listed further down.
+- colon followed by a space, mandatory; for readability.
+- short message is mandatory; explain what this PR does.
+
+Supported types are:
+- balance
+- build for code interested in building the game or build environments.
+- chore, for non-important, small changes or general updates.
+- ci for Continuous Integration; Github, builds, releases automation, also a scope.
+- doc or docs, for documentation.
+- enhance or enhancement for improving features-content.
+- feat or feature for adding-modifying features.
+- fix for bugfixes.
+- npc or factions, also a scope.
+- perf or performance, also a scope.
+- style or infrastructure or static-analysis for internal code style and infrastructure.
+- test or tests, also a scope.
+- ui for user interface related changes, also a scope.
+
+Supported scopes are:
+- accessibility
+- cbms or bionics
+- debug for debugging and troubleshoothing the game.
+- exploit
+- i18n or translation
+- maps
+- missions
+- mods
+- tools for code that is not part of the game but is part of the repo.
+
+Based on https://www.conventionalcommits.org/en/v1.0.0/
+-->
+
 #### Summary
```

Comment:
The summary section needs to stay at the top, if there is one thing the submitter reads and acts on, this must be it, and everything else is optional.